beacon-chain/rpc/beacon_server.go

```diff
@@ -86,127 +84,6 @@ func (bs *BeaconServer) CanonicalHead(ctx context.Context, req *ptypes.Empty) (*
 	return block, nil
 }
 
-// LatestAttestation streams the latest processed attestations to the rpc clients.
-func (bs *BeaconServer) LatestAttestation(req *ptypes.Empty, stream pb.BeaconService_LatestAttestationServer) error {
```

Comment:
no longer used

Reply:
what was the initial reason for using this ?

Reply:
No idea, but its functionality is almost the same as `PendingAttestations`. We really don't need this 😅